torch/_inductor/codegen/cpp.py

```diff
@@ -1419,6 +1419,12 @@ def masked(mask, body, other):
             code = BracesBuffer()
             code.writeline("[&]")
             with V.kernel.swap_buffers(code), code.indent():
+                if dtype == torch.bool and result.is_vec:
+                    # Unify blendv to use the VecMask<float, N>, since loading bool as VecMask<float> by default
+                    dtype = torch.float
+                    num_vectors = V.kernel._get_num_vectors(dtype)
+                    other_code_vec = f"at::vec::VecMask<{DTYPE_TO_CPP[dtype]}, {num_vectors}>::from({other_code})"
```

Comment:
simplify this with `_get_mask_type`?

Reply:
Thanks. Simplified with `_get_mask_type`.